flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/parser/JaninoCompiler.java

```diff
@@ -298,4 +312,33 @@ private static Java.Rvalue generateNoOperandTimestampFunctionOperation(String op
                 StringUtils.convertToCamelCase(operationName),
                 timestampFunctionParam.toArray(new Java.Rvalue[0]));
     }
+
+    private static String getTypeConvertMethod(SqlDataTypeSpec sqlDataTypeSpec) {
+        switch (sqlDataTypeSpec.getTypeName().getSimple().toUpperCase()) {
+            case "BOOLEAN":
+                return "convertBoolean";
+            case "TINYINT":
+                return "convertByte";
+            case "SMALLINT":
+                return "convertShort";
+            case "INTEGER":
+                return "convertInteger";
+            case "BIGINT":
+                return "convertLong";
+            case "FLOAT":
+                return "convertFloat";
+            case "DOUBLE":
+                return "convertDouble";
+            case "DECIMAL":
+                return "convertBigDecimal";
+            case "CHAR":
+            case "VARCHAR":
+            case "STRING":
+            case "`STRING`":
+                return "convertString";
+            default:
+                throw new ParseException(
+                        "Unsupported data type convert: " + sqlDataTypeSpec.toString());
```

Comment:
```suggestion
                throw new ParseException(
                        "Unsupported data type cast: " + sqlDataTypeSpec.toString());
```

Using cast might be better since user writes `cast` instead of `convert` keyword in YAML